nav/stepper/motion.py

```diff
@@ -18,8 +17,35 @@ def __init__(self, GPIO):
 		[0.0.0] Rishav
 			--- Created function
 	"""
-	def mov(self, dir, distance):
-		print(dir, distance)
+	def mov(self, dir, distance, GPIO):
+            for pin in self.step_mode_select:
```

Comment:
Step mode should be selected in main, not here. The motor controller should be on for the entire round.